make/clean.mk

```
@@ -31,6 +31,12 @@ clean-cluster-wide-config:
 clean-toolchain-namespaces-in-e2e:
 	$(Q)-oc get projects --output=name | grep -E "toolchain-(member|host)(\-operator)?(\-[0-9]+)?" | xargs oc delete
 
+.PHONY: clean-toolchain-dev-sso-resources
+## Delete resources in the dev sso namespace
+clean-toolchain-dev-sso-resources:
+	$(Q)-oc delete keycloak -n ${DEV_SSO_NS} --all || true
```

we may add the `--wait` flag here to avoid problems with finalizers
```suggestion
	$(Q)-oc delete keycloak -n ${DEV_SSO_NS} --all --wait || true
```

---

> also, (just curiosity) shouldn't we crash if deletion don't complete successfully?

Will it complain if there is nothing to delete? If not then I guess we can fail with an error if it failed to delete but nothing to delete is ignored.

---

I tested locally with kind, kubectl, and oc. it seems not to fail if there is nothing to delete

---

good point - addressed in https://github.com/codeready-toolchain/toolchain-e2e/pull/996/commits/316f41a00775dfcde57182ad796ab2480121a903  👍